legal-api/src/legal_api/services/filings/validations/alteration.py

```diff
@@ -97,3 +98,14 @@ def company_name_validation(filing):
                         'path': legal_name_path})
 
     return msg
+
+
+def type_change_validation(filing):
+    """Validate type change."""
+    msg = []
+    legal_type_path: Final = '/filing/business/legalType'
```

Comment:
There are 2 sections that have ```legalType```
**Metadata**: existing business info
```/filing/business/legalType```
and the **filing** data
```filing/alteration/business/legalType```
is this checking the right one?

Reply:
I changed to use the alteration path